ph5/utilities/set_deploy_pickup_times.py

```diff
@@ -65,14 +69,8 @@ def _epoch(self, epoch_l):
         self.type_s = 'BOTH'
 
 
-#
-# Read Command line arguments
-#
-
-
 def get_args():
-    global ARRAY_FILE, DEPLOY, PICKUP
-
+    global ARRAY_FILE, DEPLOY, PICKUP, AUTO_CORRECT, NICKNAME, PATH
```

Comment:
We'd like to avoid using global variables. These should be removed.

Reply:
Commits 896cee4